exercises/en/test_07_03.py

```diff
@@ -0,0 +1,27 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    # If an assertion fails, the message will be displayed
+    
+    assert 'alt.selection_interval' in __solution__, "Make sure you are creating a selection interval and storing it in a varialbe called 'brush'."
+    assert linked_scatter.encoding.x.shorthand == 'body_mass_g', "Make sure you are plotting body_mass_g on the x-axis"
+    assert not linked_scatter.encoding.x.title is None, "Make sure you are providing a title for your X-axis"
+    assert not linked_scatter.encoding.x.title.islower(), "Make sure the plot X-axis title is capitalized."
+    assert linked_scatter.encoding.x.title.count("_") == 0, "Make sure your X-axis title does not contain underscores"
+    assert linked_scatter.encoding.y.shorthand == 'flipper_length_mm', "Make sure you are plotting flipper_length_mm on the Y-axis"
+    assert not linked_scatter.encoding.y.title is None, "Make sure you are providing a title for your Y-axis"
+    assert not linked_scatter.encoding.y.title.islower(), "Make sure the plot Y-axis title is capitalized."
+    assert linked_scatter.encoding.y.title.count("_") == 0, "Make sure your Y-axis title does not contain underscores"
+    assert linked_scatter.encoding.color.condition.selection == 'selector001', "Make sure you are using the brush you defined earlier to color the points."
```

Comment:
Does this work if you name the selector something else? like use the `name=` parameter?